apps/patient/src/components/BrandedPharmacyCard.tsx

```diff
@@ -46,8 +46,8 @@ export const BrandedPharmacyCard = ({ pharmacyId, handleSelect }: Props) => {
   ) : null;
 
   return (
-    <Container>
-      <Box py={2} bgColor="white" cursor="pointer" onClick={() => handleSelect?.(pharmacyId)}>
+    <Container p={3}>
+      <Box bgColor="white" cursor="pointer" onClick={() => handleSelect?.(pharmacyId)}>
```

Comment:
What's with the style changes in this PR?

Reply:
inconsistent padding on pickup vs branded pharmacies here